src/routes/(console)/organization-[organization]/billing/budgetCap.svelte

```diff
@@ -47,72 +49,91 @@
     $: if (!capActive) {
         budget = null;
     }
+
+    let formHolderElement: HTMLDivElement;
+
+    $: if ($scrollToUpdateBudget) {
+        // clean the url in the search bar.
+        history.replaceState(null, '', $page.url.pathname);
+
+        tick().then(() => {
+            const elementTop = formHolderElement.getBoundingClientRect().top + window.scrollY;
+            // Offset 60 to prevent misalignment.
+            window.scrollTo({ top: elementTop - 60 });
+        });
+    }
```

Comment:
I'm not a fan of this workaround... too much "special" code for a small edge case and it doesn't scale well if we have the same issue in the future
Let's try to rely on the native browser behaviour :+1: 

Let's try doing what I've mentioned in the alert component, and if that works let's clean up this component